.editorconfig

```diff
@@ -15,6 +15,6 @@ trim_trailing_whitespace = true
 [*.{yml,yaml}]
 indent_size = 2
 
-[*.{[Bb][Aa][Tt],[Cc][Mm][Dd]}]
-# DOS/Win *requires* BAT/CMD files to have CRLF newlines
+[*.{[Bb][Aa][Tt],[Cc][Mm][Dd],[Rr][Ee][Gg],[Pp][Ss]1}]
```

Comment:
Then a fair question would also be here - do we need to specify handling of these file endings? CRLF is already the default, see above.

Reply:
In my experience, `.reg` and `.ps1` files can have LF line endings. Are we supporting older versions of Windows here? If so, how old? XP?

Reply:
I think Win 7 is the oldest version which is supported, right @rashil2000?

But does it matter here? We already have set CRLF for all files as the **default**. Does it make sense to additionally mention `.bat`, `.cmd`, `.reg` and `.ps1`? Seems to me like duplication...

Reply:
The oldest version we support is 7.

If we've already put a default for all text files as CRLF, then we shouldn't be special-casing extensions after that.

Reply:
Alright, removed the duplication 👍